docs/tutorials/getting-started-with-csharp/WeatherMicroservice/README.md

```diff
@@ -0,0 +1,27 @@
+C# Microservices Sample
+================
+
+This sample is created during the [Creating Microservices hosted in Docker](../microservices.html)
```

Comment:
This is just a project README that won't be published to the docs website, right?

In that case, this (and all the other links) should probably be absolute links to the website.